monkey/infection_monkey/command_builders/utils.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import PurePath
+from typing import Optional, Union
+
+from agentpluginapi import DropperExecutionMode, LinuxRunOptions, WindowsRunOptions
+
+from infection_monkey.model import DROPPER_ARG, MONKEY_ARG
+
+
+def get_agent_argument(run_options: Union[WindowsRunOptions, LinuxRunOptions]) -> str:
+    agent_arg = MONKEY_ARG
+    if run_options.dropper_execution_mode == DropperExecutionMode.DROPPER:
+        agent_arg = DROPPER_ARG
```

Comment:
This means we'll always use the dropper (i.e. dr0pp3r) when the execution mode is `DropperExecutionMode.SCRIPT`. Is that what we want? Or would we rather use `m0nk3y` in the script. What is SNMP doing now?

Reply:
When it is a `DropperExecutonMode.SCRIPT` we don't have any arguments. Check the `build_run_command`. This is going to be used only when we have anything different than `DropperExecutonMode.SCRIPT`

Reply:
> This is going to be used only when we have anything different than `DropperExecutonMode.SCRIPT`

More accurately, we **intend** that this will only be used with certain dropper modes. In other words, we've written a function where the behavior for `DropperExecutionMode.SCRIPT` is undefined. It should either raise a `ValueError` if it gets something it didn't expect and/or it should have a docstring that describes its behavior.